lib/ContactsMenu/Providers/DetailsProvider.php

```diff
@@ -60,6 +57,11 @@ public function process(IEntry $entry) {
 			return;
 		}
 
+		if ($entry->getProperty('isLocalSystemBook') === true) {
+			// Cannot show details -> ignore
+			return;
```

Comment:
Will this still show their email address though? Because that can be set in the personal details. :) The only thing which should be hidden is the details icon, not the whole entry.